src/Components/reusables/pro-upsell/ProUpsellDisplay.tsx

```diff
@@ -94,11 +95,12 @@ const ProUpsellDisplay: FunctionComponent<ProUpsellDisplayProps> = ({
 
   const fetchOfferings = async () => {
     const offerings: PurchasesOfferings = await Purchases.getOfferings();
+    console.log('offerings', offerings);
```

Comment:
one more console log here

Reply:
I think the commit after that one removed the console.log statement.